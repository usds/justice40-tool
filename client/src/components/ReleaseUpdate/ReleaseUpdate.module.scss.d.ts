declare namespace ReleaseUpdateNamespace {
    export interface IReleaseUpdateScss {
      releaseUpdateComponent: string;
      releaseUpdateContainer: string;
      releaseSectionTitle:string;
      releaseSectionBody:string;
      releaseHeader: string;
      showHideText: string;
      showHideIcon: string;
    }
  }

declare const ReleaseUpdateScssModule: ReleaseUpdateNamespace.IReleaseUpdateScss & {
    /** WARNING: Only available when "css-loader" is used without "style-loader" or "mini-css-extract-plugin" */
    locals: ReleaseUpdateNamespace.IReleaseUpdateScss;
  };

  export = ReleaseUpdateScssModule;
