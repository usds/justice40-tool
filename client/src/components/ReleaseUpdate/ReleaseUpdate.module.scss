@use '../../styles/design-system.scss' as *;

.releaseUpdateComponent{

    .releaseUpdateContainer{
        @include u-margin-top(2);
        @include u-padding(3);
        border: 2px solid #9A9B99;
        border-radius: 2px;

        .releaseHeader{
            font-size: x-large;
            font-weight: 600;
            @include u-margin-bottom(2);
        }

        .releaseSectionTitle{
            font-size: large;
            border-bottom: 2px solid #9A9B99;
        }
    }

    .showHideText {
        //remove styling of button
        border: none;
        background-color: white;

        //emulate a link
        cursor:pointer;
        text-decoration:underline;

        min-width: 45px;
        padding-left: 0;
    }

    .showHideIcon{
        vertical-align: middle;
        height: 1rem;
        filter: invert(8%) sepia(90%) saturate(7490%) hue-rotate(247deg) brightness(105%) contrast(143%);

        &:hover{
            cursor: pointer;
        }
    }
}
