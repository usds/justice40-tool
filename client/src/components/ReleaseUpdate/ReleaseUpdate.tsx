import React, {useState} from 'react';
import {useIntl} from 'gatsby-plugin-intl';

import * as styles from './ReleaseUpdate.module.scss';
import * as DOWNLOAD_COPY from '../../data/copy/downloads';

// @ts-ignore
import expandIcon from '/node_modules/uswds/dist/img/usa-icons/expand_more.svg';
// @ts-ignore
import collapseIcon from '/node_modules/uswds/dist/img/usa-icons/expand_less.svg';

export interface IReleaseUpdateProps {
}

interface IJ40AccordionItem {
  id: string,
  children: React.ReactElement
}


/**
 * This function will create the custom Accordion item. This will be used
 * for the race and age demographic UI elements
 *
 * @param {IJ40AccordionItem} props
 * @return {JSX.Element}
 */
const J40AccordionItem = ({id, children}:IJ40AccordionItem) => {
  const intl = useIntl();
  const [isExpanded, setIsExpanded] = useState(false);
  return (
    <>
      <span>
        {intl.formatMessage(DOWNLOAD_COPY.PAGE_INTRO.VIEW)}{' '}
        <a
          className={styles.showHideText ? `usa-link ${styles.showHideText}` : `usa-link`}
          href={'javascript:void(0)'}
          id={`${id}-header`}
          aria-controls={`${id}-panel`}
          // aria-expanded={isExpanded}
          onClick={() => setIsExpanded(!isExpanded)}
          tabIndex={0}
        >
          {intl.formatMessage(DOWNLOAD_COPY.PAGE_INTRO.CHANGE_LOG)}
        </a>
        { isExpanded ?
          <img
            className={styles.showHideIcon}
            src={collapseIcon}
            alt={'collapse icon'}
            onClick={() => setIsExpanded(!isExpanded)}
          /> :
          <img
            className={styles.showHideIcon}
            src={expandIcon}
            alt={'expand icon'}
            onClick={() => setIsExpanded(!isExpanded)}
          />
        }
      </span>

      <section
        id={`${id}-panel`}
        aria-labelledby={`${id}-header`}
        hidden={!isExpanded}
        className={styles.releaseUpdateContainer}
      >{children}
      </section>
    </>
  );
};

const ReleaseUpdate = ({}: IReleaseUpdateProps) => {
  return (
    <div className={styles.releaseUpdateComponent}>
      <J40AccordionItem id={'releaseUpdate'}>
        <div>

          <div className={styles.releaseHeader}>
            {DOWNLOAD_COPY.RELEASE_1_0.UPDATE_1}
          </div>

          <div className={styles.releaseSectionTitle}>
            {DOWNLOAD_COPY.RELEASE_1_0.SECTION1}
          </div>

          <div>
            <ul>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B1}</li>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2}</li>

              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_1}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_1_1}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_1_2}</li>
                </ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_2}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_1_3}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_2_2}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_2_1}</li>
                </ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_3}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_3_1}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_3_2}</li>
                </ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_4}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_4_1}</li>
                </ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_5}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B2_5_1}</li>
                </ul>
              </ul>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B3}</li>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B4}</li>
              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B4_1}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B4_2}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B4_3}</li>
              </ul>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B5}</li>
              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B5_1}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B5_2}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B5_3}</li>
              </ul>

              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6}</li>
              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_1}</li>
                  {/* <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2}</li>
                  <ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1_1}</li>
                    </ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_1}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_2}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_3}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_4}</li>
                    </ul>
                  </ul> */}
                </ul>

                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_2}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_2_1}</li>
                  {/* <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2}</li>
                  <ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1_1}</li>
                    </ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_1}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_2}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_3}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_4}</li>
                    </ul>
                  </ul> */}
                </ul>

                {/* <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_3}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2}</li>
                  <ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1_1}</li>
                    </ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_2}</li>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_4}</li>
                    </ul>
                  </ul>
                </ul>

                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_4}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2}</li>
                  <ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1_1}</li>
                    </ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2}</li>
                    <ul>
                      <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_4}</li>
                    </ul>
                  </ul>
                </ul> */}

                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5_1}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5_2}</li>
                  <ul>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5_2_1}</li>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5_2_2}</li>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5_2_3}</li>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_5_2_4}</li>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_1}</li>
                    <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2_2}</li>
                  </ul>
                </ul>
              </ul>

              {/* <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7}</li>
              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_1}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7_1_1}</li>
                </ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B6_1_2_2}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7_2_1}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7_2_2}</li>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7_2_3}</li>
                </ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7_3}</li>
                <ul>
                  <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B7_3_1}</li>
                </ul>
              </ul> */}

              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B8}</li>
              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B8_1}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B8_2}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B8_3}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B8_4}</li>
              </ul>

              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B9}</li>
              <ul>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B9_1}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B9_2}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B9_3}</li>
                <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B9_4}</li>
              </ul>

              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION1_B10}</li>
            </ul>
          </div>

          <div className={styles.releaseSectionTitle}>
            {DOWNLOAD_COPY.RELEASE_1_0.SECTION2}
          </div>

          <div>
            <ul>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION2_P1}</li>
              <li>{DOWNLOAD_COPY.RELEASE_1_0.SECTION2_P2}</li>
            </ul>
          </div>

          <div>
            {DOWNLOAD_COPY.RELEASE_1_0.FOOTER}
          </div>

        </div>
      </J40AccordionItem>
    </div>
  );
};

export default ReleaseUpdate;
