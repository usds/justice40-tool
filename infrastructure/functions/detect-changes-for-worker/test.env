REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1