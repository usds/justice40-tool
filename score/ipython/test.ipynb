{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
