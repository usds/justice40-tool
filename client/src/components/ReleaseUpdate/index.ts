import ReleaseUpdate from './ReleaseUpdate';
export default ReleaseUpdate;
