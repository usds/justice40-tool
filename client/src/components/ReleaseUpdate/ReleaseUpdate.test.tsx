import React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import ReleaseUpdate from './ReleaseUpdate';

describe('rendering of ReleaseUpdate Component', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <ReleaseUpdate />
      </LocalizedComponent>,
  );
  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
