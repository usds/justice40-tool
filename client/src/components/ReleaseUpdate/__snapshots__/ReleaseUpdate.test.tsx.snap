// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of ReleaseUpdate Component checks if component renders 1`] = `
<DocumentFragment>
  <div>
    <span>
      View 
      <a
        aria-controls="releaseUpdate-panel"
        class="usa-link"
        href="javascript:void(0)"
        id="releaseUpdate-header"
        tabindex="0"
      >
        release notes
      </a>
      <img
        alt="expand icon"
        src="test-file-stub"
      />
    </span>
    <section
      aria-labelledby="releaseUpdate-header"
      hidden=""
      id="releaseUpdate-panel"
    >
      <div>
        <div>
          Version 1.0 Release update - Nov 22, 2022
        </div>
        <div>
          New & improved
        </div>
        <div>
          <ul>
            <li>
              Added lands within the boundaries of Federally Recognized Tribes and 
    locations of Alaska Native Villages using data from the Bureau of Indian
    Affairs at the U.S. Department of the Interior
            </li>
            <ul>
              <li>
                Added new data for indicators of burden
              </li>
              <li>
                Climate change
              </li>
              <ul>
                <li>
                  Projected flood risk
                </li>
                <li>
                  Projected wildfire risk
                </li>
              </ul>
              <li>
                Housing
              </li>
              <ul>
                <li>
                  Historic underinvestment due to redlining
                </li>
                <li>
                  Lack of green space
                </li>
                <li>
                  Lack of plumbing
                </li>
              </ul>
              <li>
                Legacy pollution
              </li>
              <ul>
                <li>
                  Abandoned mine lands
                </li>
                <li>
                  Formerly used defense sites
                </li>
              </ul>
              <li>
                Transportation
              </li>
              <ul>
                <li>
                  Transportation barriers
                </li>
              </ul>
              <li>
                Water and wastewater
              </li>
              <ul>
                <li>
                  Underground storage tanks and releases
                </li>
              </ul>
            </ul>
            <li>
              Included communities that are completely surrounded by other 
    disadvantaged communities and that meet an adjusted low income 
    threshold
            </li>
            <li>
              Made technical changes to enhance accuracy of the tool
            </li>
            <ul>
              <li>
                Removed income data for students enrolled in higher education in the low income indicator
              </li>
              <li>
                Imputed income for census tracts missing income data
              </li>
              <li>
                Used transportation barriers and expected population loss rate 
    data burdens only for census tracts with populations of 20 or 
    more to exclude areas such as airports
              </li>
            </ul>
            <li>
              Improved the user interface
            </li>
            <ul>
              <li>
                Added demographics (race & age) for census tracts to map side panel
              </li>
              <li>
                Improved the design of the map side panel
              </li>
              <li>
                Updated the site copy of the website
              </li>
            </ul>
            <li>
              Added new data for all the U.S. Territories, and also made a small methodology change for Puerto Rico
            </li>
            <ul>
              <li>
                USVI
              </li>
              <ul>
                <li>
                  Included data from 2010 decennial census for US Virgin Islands
                </li>
              </ul>
              <li>
                Guam
              </li>
              <ul>
                <li>
                  Included data from 2010 decennial census for Guam
                </li>
              </ul>
              <li>
                Puerto Rico
              </li>
              <ul>
                <li>
                  Removed linguistic isolation as an indicator in the methodology for Puerto Rico
                </li>
                <li>
                  Imported additional available data for Puerto Rico
                </li>
                <ul>
                  <li>
                    Energy cost
                  </li>
                  <li>
                    Housing cost
                  </li>
                  <li>
                    Abandoned mine lands
                  </li>
                  <li>
                    Proximity to hazardous waste sites
                  </li>
                  <li>
                    Proximity to Superfund (National Priorities List (NPL)) sites
                  </li>
                  <li>
                    Proximity to Risk Management Plan (RMP) sites
                  </li>
                </ul>
              </ul>
            </ul>
            <li>
              Enhanced the technical files:
            </li>
            <ul>
              <li>
                Added all new data indicators and demographics to .xls, .csv, and shapefiles
              </li>
              <li>
                Added previous versions page to access the beta versionâ€™s 
    download files
              </li>
              <li>
                Updated codebook
              </li>
              <li>
                Updated Technical Support Document
              </li>
            </ul>
            <li>
              Improved the way that users provide feedback on the tool:
            </li>
            <ul>
              <li>
                Data survey
              </li>
              <li>
                Site experience survey
              </li>
              <li>
                Census tract feedback
              </li>
              <li>
                General contact form
              </li>
            </ul>
            <li>
              Added a link to sign up for email mailing list managed by CEQ
            </li>
          </ul>
        </div>
        <div>
          Fixes
        </div>
        <div>
          <ul>
            <li>
              Bug fix: loaded missing life expectancy data for Maine and Wisconsin
            </li>
            <li>
              Bug fix: Census tracts that are 100% water should not be 
    included on the map
            </li>
          </ul>
        </div>
        <div>
          release version 1.0
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
